sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlDialectSpecTest.java

```diff
@@ -4114,4 +4114,85 @@ public void testCountIfZetaSQLDialect() {
 
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
+
+  @Test
+  public void testLogicalAndZetaSQL() {
```

Comment:
testLogicalAndZetaSQL fails because of  [BEAM-9514]( https://issues.apache.org/jira/browse/BEAM-9514)